store.go

```diff
@@ -211,6 +211,31 @@ func (s *Store) MustFind(q Query) *ResultSet {
 	return rs
 }
 
+// Reload refreshes the record with the data in the database and makes the
+// record writable.
+func (s *Store) Reload(record Record) error {
+	if record.GetID().IsEmpty() {
```

Comment:
are you proposing (a) to rename the method, or (b) to access the property by name?
a) can not be done because ID is alredy used for the model property named `ID`
b) can not be accessed using .ID because Record is not an struct but an interface